edx_proctoring/templates/proctored_exam/confirm-decline.html

```diff
@@ -8,7 +8,7 @@
     </div>
     <div class="msg-content">
       {% blocktrans %}
-        If you take this exam without proctoring, you will <strong> no longer be eligible for academic credit. </strong>
+        If you take this exam without proctoring, you will not be eligible for course credit or the MicroMasters credential if either applies to this course.
```

Comment:
@marcotuts Do we want to specifically refer only to "MicroMasters", or should we be more generic? e.g. "you will not be eligible for course credit or credit towards program completion."